### PR TITLE
[KOGITO-2700] Tests: increase timeout and concurrent

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-2700

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/framework/namespace.go [hunk 1]
```diff
@@ -70,6 +70,7 @@ func OperateOnNamespaceIfExists(namespace string, operate func(namespace string)
 	} else if ok {
 		return operate(namespace)
 	}
+	GetLogger(namespace).Warn("No namespace found for deletion")
```
Comment (sutaakar) [inline test/framework/namespace.go:73]:
```suggestion
	GetLogger(namespace).Warn("Namespace not found for operation")
```

File: test/framework/namespace.go [hunk 2]
```diff
@@ -70,6 +70,7 @@ func OperateOnNamespaceIfExists(namespace string, operate func(namespace string)
 	} else if ok {
 		return operate(namespace)
 	}
+	GetLogger(namespace).Warn("Namespace not found for operation")
```
Comment (MarianMacik) [inline test/framework/namespace.go:73]: Maybe even better would be to name which namespace was not found, but maybe that is too much. :)

Comment (radtriste) [reply test/framework/namespace.go:73]: well namespace is in the logger already `GetLogger(namespace)`

Comment (MarianMacik) [reply test/framework/namespace.go:73]: True :)

Comment (sutaakar) [reply test/framework/namespace.go:73]: Well, namespace name is already reflected in logger name

**Pull request merged**